package.json

```diff
@@ -247,7 +249,7 @@
     "babel-plugin-transform-remove-console": "^6.9.4",
     "clean-webpack-plugin": "^3.0.0",
     "concurrently": "^5.3.0",
-    "copy-webpack-plugin": "^6.4.1",
+    "copy-webpack-plugin": "^10.1.0",
```

Comment:
Please make sure this change won't create regressions

Reply:
I tested if the app builds and runs properly on all platforms, and it did, do you think there is something else I should do?

Reply:
I think it's enough